src/Rose/AST/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 add_library(RoseAST OBJECT
   graphviz.C
-  io/merge.C io/link.C io/delete.C io/share.C io/load.C
+  #io/merge.C io/link.C io/delete.C io/share.C io/load.C
```

Comment:
If these configurations are not needed anymore, I think they could be removed instead of commented out.